src/globus_sdk/experimental/tokenstorage_v2/base.py

```diff
@@ -0,0 +1,129 @@
+from __future__ import annotations
+
+import abc
+import contextlib
+import os
+import typing as t
+
+from globus_sdk.services.auth import OAuthTokenResponse
+
+
+class StorageAdapterV2(metaclass=abc.ABCMeta):
```

Comment:
Would it make sense to rename it to something more specific with this change?

`TokenStorageAdapterV2` or `TokenStorage` or something?

The interface `StorageAdapter` is was more generic in name than the actual class's scope. It implies with such a generic name that we're not gonna be storing anything else (e.g. consents)

Reply:
`TokenStorage` seemed the most accurate to me